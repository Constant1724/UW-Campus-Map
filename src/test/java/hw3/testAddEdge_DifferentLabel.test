# START_TEST
# NAME DifferentLabel
# test if we can add edges to the graph.
# Edges will have same start and end but different labels.
# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 ValidNode1
AddNode graph1 ValidNode2

# Add an edge
AddEdge graph1 ValidNode1 ValidNode2 label1
AddEdge graph1 ValidNode1 ValidNode2 label2
# END_TEST