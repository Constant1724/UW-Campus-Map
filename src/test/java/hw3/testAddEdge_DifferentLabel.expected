# START_TEST
# NAME DifferentLabel
# test if we can add edges to the graph.
# Edges will have same start and end but different labels.
# Create a graph
created graph graph1

# Add a pair of nodes
added node ValidNode1 to graph1
added node ValidNode2 to graph1

# Add an edge
added edge label1 from ValidNode1 to ValidNode2 in graph1
added edge label2 from ValidNode1 to ValidNode2 in graph1
# END_TEST